lib/cWarnings.mli

```diff
@@ -19,3 +19,5 @@ val set_flags : string -> unit
 (** Cleans up a user provided warnings status string, e.g. removing unknown
     warnings (in which case a warning is emitted) or subsumed warnings . *)
 val normalize_flags_string : string -> string
+
+val pr_warnings : category_name:(string * string option) option -> Pp.std_ppcmds
```

Comment:
This seems odd. I'd have expected three functions: `pr_warning`, `pr_warning_category`, `pr_all_warnings`, or one function taking an argument `All | Warning of string | Category of string". Why can't the user type only the warning name? Also, it would remove the need for the strange error "xx does not belong to category yy".